src/server/views/dashboard/lists-edit.njk

```diff
@@ -1,14 +1,14 @@
 {% extends "./dashboard.njk" %}
 
```

Comment:
Per the trello card (https://trello.com/c/ZbJBJmdL/1490-annual-review-list-settings-annual-review-start-date#comment-6351a979702f6600e74e9b8e) this page also needs a back link at the top and bottom of the page, similar to the list item details page.